style.css

```diff
@@ -1,60 +1,63 @@
-*{
-    margin: 0;
-    padding: 0;
-    color: #fff !important;
+* {
+  margin: 0;
+  padding: 0;
+  color: #fff !important;
 }
-*{
-    margin: 0;
-    padding: 0;
-    color: #fff !important;
-}
-.navbar{
-    background-color: #BE3144;
-}
-.navbar .navbar-nav a{
-    font-size: 20px;
```

Comment:
Please consider removing !important and consider fixing .navbar .navbar-nav a :weary: